client/src/app/_models/Barman.ts

```diff
@@ -28,6 +28,10 @@ export class Barman {
     roles?: AssociationChanges,
   };
 
+  public isActive(): boolean {
+    return this.leaveAt ? false : true;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Simplify this expression. ([typescript:S1125](https://sonarcloud.io/organizations/k-fet/rules#rule_key=typescript%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=k-app&pullRequest=302&issues=AWdH7aJv1-jKZJbrex9i&open=AWdH7aJv1-jKZJbrex9i)

[ISSUE_KEY]: AWdH7aJv1-jKZJbrex9i